controllers/comments_controller.go

```diff
@@ -0,0 +1,47 @@
+package controllers
+
+import (
+	"time"
+	"encoding/json"
+)
+
+type CommentsController struct {
```

Comment:
@breakingrobot For a class named **Comments**Controller, **that's pretty ironic that it isn't commented**.